requirements.txt

```diff
@@ -10,3 +10,4 @@ mock==1.3.0
 # remove this upper bound on the wheel version once 2.6 support
 # is dropped from aws-cli
 wheel>0.24.0,<0.30.0
+distro; python_version>="3.8"
```

Comment:
We're very weary against pulling in additional dependencies.  Is there no way to do this using just the stdlib?